lib/msal-node/README.md

```diff
@@ -0,0 +1,33 @@
+# TSDX Bootstrap
```

Comment:
You should copy the file from MSAL browser and make updates to it here. You can even remove all the text, but just keep the formatting to make it easier. That way we have the same templates across all platforms.

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/1332 is raised to track this for accuracy. Can we please limit this PR for node scaffolding only?